### ISSUE TITLE
 XNNPACK disabled in build

### ISSUE THREAD

**Issue by StuartIanNaylor** [42io/tflite_kws#1]
Is there any reason for XNNPACK cmake build being disabled as would of thought with TFlite and embedded linux delegate that would of been the best option.
Is it disabled purely for some sort of legacy or is there a reason?

Ps the full release of 1.7.1 seems to work fine 1.8.0 not so.

On another note but guess this is up to individual choice the accuracy loss of quantisation is very minimal with TFLite.
https://www.tensorflow.org/lite/performance/post_training_quantization

One thing though after training to save signature parameters as usage data could be collect for post training on device.
https://www.tensorflow.org/lite/examples/on_device_training/overview
Never tried it and not sure how much data and epochs you could run or maybe you need to blend with original data to stop overfitting.
